odoo/tools/image.py

```diff
@@ -75,7 +75,7 @@ def image_resize_image(base64_source, size=(1024, 1024), encoding='base64', file
 
     if image.size != size:
         image = image_resize_and_sharpen(image, size)
-    if image.mode not in ["1", "L", "P", "RGB", "RGBA"]:
+    if image.mode not in ["1", "L", "P", "RGB"]:
```

Comment:
Hmm that would discard the alpha channel for all images, even when saving in a format that does support it (e.g. PNG). We'll probably need to be more selective than that for the conversion.
Do you have the error trace so we can figure out why we want to convert RGBA sources to JPEG?